extensions.json

```
@@ -1169,6 +1169,11 @@
       "version": "1.2.6",
       "checkout": "v1.2.6"
     },
+    {
+      "id": "vinliao.fast-arrow",
+      "repository": "https://github.com/vinliao/fast-arrow",
+      "version": "1.0.8"
```

No need to pin a specific version if we publish straight from the default branch.

```suggestion
      "repository": "https://github.com/vinliao/fast-arrow"
```